src/chains/ethereum.ts

```diff
@@ -35,14 +36,29 @@ export class NearEthAdapter {
     derivationPath: string;
     sender: Address;
   }) {
-    this.client = createPublicClient({ transport: http(config.providerUrl) });
+    this.ethClient = createPublicClient({
+      transport: http(config.providerUrl),
+    });
     this.scanUrl = config.scanUrl;
     this.mpcContract = config.mpcContract;
     this.gasStationUrl = config.gasStationUrl;
     this.derivationPath = config.derivationPath;
     this.sender = config.sender;
   }
 
+  /**
+   * @returns ETH address derived by Near account via `derivationPath`.
+   */
+  ethPublicKey(): Address {
+    return this.sender;
+  }
+  /**
+   * @returns Near accountId linked to derived ETH.
+   */
+  nearAccountId(): string {
+    return this.mpcContract.contract.account.accountId;
+  }
```

Comment:
These public access methods were recently made while making this.sender private and the opensea script got botched because it still used this field.

Script was hotfixed [here](https://github.com/Mintbase/near-ca/commit/02440333c7f564f7117d224c2f09b8253ec13f4f), but we should really run some CI to ensure that the examples directory is "sound"